cove_360/templates/cove_360/additional_checks.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'base.html' %}
+{% load i18n %}
+{% block content %}
+
+<div>
+  <h2>{% trans "Additional Checks" %}</h2>
+
+<p>
+
+This page details all of the Additional Checks in this instance of the Data Quality Tool. 
+
+The <label> Heading </label> is prefixed by text such as "X of your grants have..." in almost all cases, apart from when the check refers to the whole dataset (eg the multiple Funding Org check). The <label>Message</label> is displayed if the check is triggered. The <label>Method</label> describes how the check works under-the-hood. 
+</p>
+
+<h2> Checks </h2>
+{% for check in checks %}
+
+<p>
+<label>Heading</label> {{ check.heading }}</br>
```

Comment:
Templates in Cove do not have spaces between the curly braces and the variable name, so ```{{check.heading}}``` instead of ```{{ check.heading }}```. I personally prefer the latter, but not using spaces is more consistent  with the rest of the code.